argus/backend/controller/testrun_api.py

```diff
@@ -289,6 +289,22 @@ def ignore_jobs():
     }
 
 
+@bp.route("/resolve_runs", methods=["POST"])
+@api_login_required
+def resolve_runs():
+    payload = get_payload(request)
```

Comment:
Maybe instead generic `payload` use `test_id_test_run_tuples_list` showing what payload should contain?

Reply:
Added a type hint since this one is just an array.